docs/_articles/en/einstein/einstein-overview.md

```diff
@@ -5,19 +5,21 @@ lang: en
+Einstein for Developers is an AI-powered developer tool that's available as an easy-to-install Visual Studio Code extension built using CodeGen, the secure, custom AI model from Salesforce. The extension is available in the [VS Code](https://marketplace.visualstudio.com/vscode) and [Open VSX](https://open-vsx.org/) marketplaces.
 
 Einstein for Developers assists you throughout the Salesforce development process with expertise learned from anonymized code patterns. Our suite of AI-powered developer tools increases productivity and provides helpful assistance for complex coding tasks. We enforce development best practices with code generation and our suite of recommended static analysis and security scanning tools. With boilerplate code generation as its foundation, AI-assisted tooling also makes it easier for new developers to onboard to the Salesforce Platform.
 
 **Important**: This feature is a Beta Service. A customer may opt to try such Beta Service in its sole discretion. Any use of the Beta Service is subject to the applicable Beta Services Terms provided at [Agreements and Terms](https://www.salesforce.com/company/legal/agreements/).
 
 ## Current Capabilities
 
-This release of Einstein for Developers focuses on boilerplate Apex code generation from natural language prompts. This feature, used along with IntelliSense, makes Apex development tooling in Visual Studio Code even richer. Familiarity with Visual Studio Code is assumed.
+Einstein for Developers generates Apex code from natural language prompts and automatically suggests code completions for you as you type. When enabled along with IntelliSense, this feature makes Salesforce development tooling in Visual Studio Code even richer. Familiarity with Visual Studio Code is assumed.
 
-Use the extension in this release to generate boilerplate code from natural language instructions in a sidebar, so you can work with your editor and the tool side by side, without any interruptions to your workflow. You can also get code suggestions within an existing Apex class, trigger, or anonymous Apex file. Use the VS Code Command Palette to enter a prompt describing what you'd like to build and then generate code suggestions within your editor.
+* Enter natural language instructions in a sidebar, so you can work with your editor and the tool side by side, without any interruptions to your workflow. Or use the VS Code Command Palette to enter a prompt describing what you'd like to build and then generate code suggestions within your editor.
+* Enable inline autocompletion to automatically receive suggestions as you write Apex and LWC code.
```

Comment:
It's enabled by default.  This kind of felt like a call to action

Reply:
changed to "use"